js/content/store.js

```diff
@@ -455,6 +455,25 @@ class StorePageClass {
             })
         });
     }
+
+    forceVideoMP4() {
+        if (!SyncedStorage.get("mp4video")) { return; }
+
+        Array.from(document.querySelectorAll("[data-webm-source]")).forEach(function(node) {
```

Comment:
You don't need to convert a `NodeList` into an `Array` if you just want to iterate over it with `forEach`